python/cudf/cudf/utils/applyutils.py

```diff
@@ -76,7 +78,7 @@ def apply_chunks(df, func, incols, outcols, kwargs, chunks, tpb):
     {params_chunks}
     """
     applyrows = ApplyChunksCompiler(
-        func, incols, outcols, kwargs, cache_key=None
+        func, incols, outcols, kwargs, pessimistic_nulls=False, cache_key=None
```

Comment:
Is the choice to not expose this to people using `apply_chunks` intentional?

Reply:
I have not familiarize myself with apply_chunks yet. It looks to be `apply_rows` but with custom kernel block size? If so, we should the pessimistic flag there as well.

Reply:
Sounds good to me. That is my general understanding, too.

Reply:
added to apply_chunks.

Reply:
Oof. I almost forgot to remove the `=False`. :1st_place_medal: